docs/reference/packages.rst

```diff
@@ -14,12 +14,12 @@ packages:
 
 1. Python modules and packages are turned into distribution package
    archives, and published at the Python Package Index (PyPI).
-2. `upip` package manager can be used to install a distribution package
-   on a `MicroPython port` with networking capabilities (for example,
-   on the Unix port).
+2. `upip` package manager is used to install a distribution package
+   on a `MicroPython port` with networking capabilities. You can run upip
```

Comment:
Not sure this needs a rewrite: the previous text conveyed the same information (apart from explicitly mentioning some microcontroller ports are example of ports with network capabilities), but now there are just 2 sentences which essentially say the same thing.

How about adding the microcontroller bit to the original and explicitly saying this requires an internet connection (since AFAIK it only works against PyPy), something like

```
The `upip` package manager can be used to install a distribution package
on any `MicroPython port` with networking capabilities and internet access,
for example the Unix port and microcontroller ports like ESP8266.
```